src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/UpdateProjectAction.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.ProjectSystem;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Razor.Workspaces.ProjectSystem;
+
+internal interface IUpdateProjectAction
+{
+}
+
+internal record RemoveDocumentAction(HostDocument OriginalDocument) : IUpdateProjectAction
+{
+}
```

Comment:
Consider using `;` rather than empty curly braces for the records defined here.

```suggestion
internal record RemoveDocumentAction(HostDocument OriginalDocument) : IUpdateProjectAction;
```